### ISSUE TITLE
MtxOption does not render & symbol

### ISSUE THREAD

**Issue by Keolan-MFOL** [ng-matero/extensions#347]
Thanks for the repository
When using mtx-option in the mtx-select, the & symbol renders as '&amp'
This seems to only affect the & symbol as other symbols render as expected.

I suspect the issue is caused in line 543 of select.ts using the innerHTML as the value for the label
line 543: $ngOptionLabel: option.elementRef.nativeElement.innerHTML, 

HTML
```html
<mat-form-field>
  <mat-label>Choose an option</mat-label>
  <mtx-select>
    @for (food of items; track food) {
      <mtx-option [value]="food.id">
        {{food.name}}
       </mtx-option>
    }
  </mtx-select>
</mat-form-field>
```

TS
```ts
  items = [
    { id: 1, name: 'Apple &🎅' },
    { id: 2, name: 'Lemon &🕵️‍♀️' },
    { id: 3, name: 'Lime !@#$%^&*()-=' },
    { id: 4, name: 'Orange' },
    { id: 5, name: 'Strawberry' },
  ];
```
Expected
![expected](https://github.com/user-attachments/assets/d7a66b61-4a4b-4440-b263-5244c8f1aaac)
Current
![current](https://github.com/user-attachments/assets/1b0828d8-705f-47e0-aea9-0dc943ed257d)

**Comment by nzbin** [comment]
<img width="239" alt="image" src="https://github.com/user-attachments/assets/60d3475f-d0f6-47b6-8a5a-c0b7dfdda26e">

I don't find any issues. Can you provide an online demo?

**Comment by Keolan-MFOL** [comment]
I've recreated the issue in [stackblitz](https://stackblitz.com/edit/stackblitz-starters-bn4tlf?file=src%2Fapp%2Fapp.component.ts)

**Comment by nzbin** [comment]
18.2.2 has fixed, please have a try.

**Comment by Keolan-MFOL** [comment]
The issue has been resolved in 18.2.2. Thank you.
